.github/workflows/dart.yml

```diff
@@ -0,0 +1,330 @@
+# Created with package:mono_repo v6.4.2
```

Comment:
+1 to having standardized tooling to support mono repos, but I do think that we need to do some additional work on the output of mono_repo. The hand-written workflow file had advantages over the machine generated one in readability and terseness (7x smaller).

Reply:
we could ABSOLUTELY have a "simple" mode. We try to do a LOT of funky work to merge flows and cache, etc – that makes things hard to read!